packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-dialog.tsx

```diff
@@ -30,6 +40,7 @@ const ContractTypeDialog = ({
                 <span className='contract-type-widget__select-arrow' />
                 <MobileDialog
                     portal_element_id='modal_root'
+                    // @ts-expect-error The type of the title prop in MobileDialog is wrong and it should be React.ReactElement
```

Comment:
@henry-deriv Please update the MobileDialog component props types accordingly instead of this comment. It's necessary. Please also kindly remember to check if no TS errors appear after your update in other places where MobileDialog is used.